package.json

```diff
@@ -85,6 +85,8 @@
     "CODE_OF_CONDUCT.md",
     "CONTRIBUTING.md",
     "dist/css/",
-    "dist/js/"
+    "dist/js/",
+    "dist/scss",
+    "dist/assets"
```

Comment:
If the goal is to include the `scss` files, we probably don't need to have the `assets`. From what I understand, Storybook uses the `assets`, which include large image files (e.g., broadway_tower.jpg) that aren't essential to this project.
```suggestion
```

Reply:
There is just [one place here](https://github.com/creativecommons/chooser/blob/main/src/components/FooterSection.vue#L16) where an svg is directly imported from the vocabulary package, which is why I included it.

Should I go ahead and remove the `assets` folder anyway? @brylie

Reply:
Updated to have just `logos`, we can update stuff later yep!